src/test/fuzz/util.h

```diff
@@ -10,6 +10,7 @@
 #include <attributes.h>
 #include <chainparamsbase.h>
 #include <coins.h>
+#include <compat.h>
```

Comment:
nit: I am not really a fan of turning this into a "include all headers that exist" file. This could be a separate module, but as a start, what about moving the implementation details to the cpp file at least?

Reply:
I agree, but note that `compat.h` was already transitively included via at least four other already included files (`net.h`, `netaddress.h`, `netbase.h` and `util/time.h`).

Thus the inclusion doesn't change anything in practice from a technical perspective: it only makes it clear that we depend on `compat.h` :)